tensorflow/python/ops/math_ops.py

```diff
@@ -568,8 +572,8 @@ def angle(input, name=None):
   For example:
 
   ```
-  # tensor 'input' is [-2.25 + 4.75j, 3.25 + 5.75j]
-  tf.angle(input) ==> [2.0132, 1.056]
+  # tensor 'input' is [-2.25 + 4.75j, 3.25 + 5.75j], in `complex64` format
+  tf.angle(input) ==> array([2.0131705, 1.056345 ], dtype=float32)
```

Comment:
And convert this to a comment (I get `float64`).

```suggestion
  tf.angle(input).numpy()
  # ==>  array([2.0131705, 1.056345 ], dtype=float64)
```

Reply:
Depending on the dtype the constant `input`, I either get `array([2.01317055, 1.05634501])` (`input` is of dtype `complex128`) or `array([2.0131705, 1.056345 ], dtype=float32)` (`input` is of dtype `complex64`). The commit I just pushed has a matching set of runnable code and exact output for the complex64 case.